prov/verbs/src/verbs_info.c

```diff
@@ -902,6 +960,8 @@ static int fi_ibv_get_srcaddr_devs(struct fi_info **info)
 	}
 
 	for (fi = *info; fi; fi = fi->next) {
+		if (fi->ep_attr->type == FI_EP_DGRAM)
+			continue;
```

Comment:
never mind my previous comment -- it looks like addressing is handled further down in the patch